src/internal/privateTypes.ts

```diff
@@ -158,7 +159,7 @@ interface ExposedNativeMethods {
 }
 
 export interface DeviceInfoNativeModule
-  extends NativeConstants,
+  extends NativeModule,
```

Comment:
Why was it changed?

Reply:
So that it can be passed to `NativeEventEmitter`